src/main/java/frc/robot/commands/CenterOnGamePiece.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commands;
+
+import com.spikes2212.command.drivetrains.commands.DriveArcadeWithPID;
+import frc.robot.services.VisionService;
+import frc.robot.subsystems.Drivetrain;
+
+public class CenterOnGamePiece extends DriveArcadeWithPID {
+
+    private final VisionService vision;
+    private final int pipeline;
+
+    public CenterOnGamePiece(Drivetrain drivetrain, VisionService vision, int pipeline) {
```

Comment:
instead of giving it a pipeline, give it a game piece (enum) and get the pipeline from that